hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/JetGetJobIdsCodec.java

```diff
@@ -89,34 +77,11 @@ public static ClientMessage encodeRequest(@Nullable java.lang.String onlyName, l
         ClientMessage.ForwardFrameIterator iterator = clientMessage.frameIterator();
         RequestParameters request = new RequestParameters();
         ClientMessage.Frame initialFrame = iterator.next();
-        if (initialFrame.content.length >= REQUEST_ONLY_JOB_ID_FIELD_OFFSET + LONG_SIZE_IN_BYTES) {
```

Comment:
I bumped the since version of the codec itself, as all of its parameters were having version `2.3`, while the codec was having `2.0`. On second thought, there is nothing wrong with having such a definition. So, I reverted this back. 

There was a problem with binary compatibility templates for such definitions, I fixed that in the protocol repo and regenerated the codecs and binary compatibility tests. See https://github.com/hazelcast/hazelcast-client-protocol/pull/385/commits/2d6c2b73dd8404db8519c6ed5ee071148f45a988